scripts/check_doc_ownership.py

```
@@ -0,0 +1,85 @@
+Usage: check_doc_ownership.py <base-rev> <head-rev>
+"""
+
+import re
+import subprocess
+import sys
+
+FN = re.compile(r"^\s*(?:pub(?:\([^)]*\))?\s+)?(?:default\s+)?(?:const\s+)?(?:async\s+)?(?:unsafe\s+)?(?:extern\s+\"[^\"]*\"\s+)?fn\s+([A-Za-z_]\w*)")
+
+
+def git(*args):
+    return subprocess.run(["git", *args], capture_output=True, text=True).stdout
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)